source/fe/mapping.cc

```diff
@@ -61,6 +61,29 @@ Mapping<dim, spacedim>::get_center(
 
 
 
+template <int dim, int spacedim>
+BoundingBox<spacedim>
+Mapping<dim, spacedim>::get_bounding_box(
+  const typename Triangulation<dim, spacedim>::cell_iterator &cell) const
+{
+  if (preserves_vertex_locations())
+    return cell->bounding_box();
+  else
+    {
+      const auto      vertices = get_vertices(cell);
+      Point<spacedim> p0 = vertices[0], p1 = vertices[0];
+      for (unsigned int j = 1; j < vertices.size(); ++j)
+        for (unsigned int d = 0; d < spacedim; ++d)
+          {
+            p0[d] = std::min(p0[d], vertices[j][d]);
+            p1[d] = std::max(p1[d], vertices[j][d]);
+          }
```

Comment:
It would be so much nicer if we had a constructor for the bounding box class that just taka a list of points